README.md

```diff
@@ -1,50 +1,107 @@
-Updated based on above debug commands and current ID template for the README.
-@ericabr please review for any missing content/improvements:
-
 # ibm-commonui-operator
 
-You can use the ibm-commonui-operator to install the Common Web UI service for the IBM Cloud Platform Common Services and access the Common Web UI console.
+> **Important:** Do not install this operator directly. Only install this operator using the IBM Common Services Operator. For more information about installing this operator and other Common Services operators, see [Installer documentation](http://ibm.biz/cpcs_opinstall).
+> If you are using this operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak to learn more about how to install and use the operator service. For more information about IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
 
-You can use the Common Web UI console to access information and features from other IBM Cloud Platform Common Services or IBM Cloud Paks that you install.
+You can use the ibm-commonui-operator to install the Common Web UI service for the IBM Cloud Platform Common Services and access the Common Web UI console. You can use the Common Web UI console to access information and features from other IBM Cloud Platform Common Services or IBM Cloud Paks that you install.
 
-For more information about the available IBM Cloud Platform Common Services and IBM Cloud Paks, see the [IBM Knowledge Center](http://ibm.biz/cpcsdocs).
+For more information about the available IBM Cloud Platform Common Services, see the [IBM Knowledge Center](http://ibm.biz/cpcsdocs).
 
 ## Supported platforms
 
-Red Hat OpenShift Container Platform 4.2 or newer installed on one of the following platforms:
+- Red Hat OpenShift Container Platform 4.2 or newer installed on one of the following platforms:
```

Comment:
The sub-bullets for the Red Hat OpenShift Container Platform 4.2 or newer.... needs to be indented.

Reply:
It goes against the linter

Reply:
Can the bullet point before prerequisite be remove (just have it for list items)